### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Praveenahs**
The purpose of this work is to design a real-time facial expression classification the same system using Deep Convolutional Neural Networks (DCNN) that could be used for security checks in public places. The design of the system is conducted in five stages. Initially, facial expression images for the seven categories are captured, preprocessed and loaded into the system, so that there are 10 images for each of the category namely angry, disgust, sad, fear, neutral, happy and surprise. The network architecture is then defined by repeating convolution layer, batch normalization layer, rectified linear layer(ReLU), max pooling layer three times. A fully connected layer is then defined for training followed by a SoftMax layer. The training process is executed followed by classification of a facial features into their appropriate label or category. When the training data of different persons a F1 score of 70.71 was obtained. When the training set from the same person, real time classification resulted in a F1 score of 85.6. The results reveal that the accuracy can be increased if the training dataset from the same person was used. Also, if the training data was increased accuracy can be further increased. The limitation was that raw data was used and can be improved by using data augmentation techniques to remove redundant data. The harmonic mean of real-time facial expression classification using the proposed word was considerably high and is suitable for applications like robotics, education, artificial intelligence and security.
Keywords: DCNN, Facial expression classification (FEC), convolution, ReLU, data augmentation, batch normalization

### CODE REVIEW

**Comment by Praveenahs** [comment]
A simple Description
